content/blog/2025/02/2025-02-14-building-a-safer-matrix.md

```diff
@@ -190,40 +190,40 @@ Rechenschaftspflicht ist wichtig, deshalb wollen wir dieses Jahr mit der Veröff
+**Unsere Werkzeuge als Open Source freigeben**  
+Während wir die Werkzeuge für das Schutzmanagement auf Matrix.org verbessern, wollen wir unsere Arbeit mit anderen teilen. Wir werden in zukünftigen Beiträgen mehr über unsere Ziele sprechen. Als einer der Partner von [ROOST](https://roost.tools/partnerships) beteiligen wir uns auch an der Arbeit an Open-Source-Schutzwerkzeugen.
 
 **Erfüllung und Übertreffen unserer Verpflichtungen gemäß dem Online Safety Act**  
 Wir sind im Vereinigten Königreich ansässig und haben uns von Anfang an produktiv mit dem Online Safety Act auseinandergesetzt. Dazu gehört auch, dass wir uns weiterhin entschieden gegen die Bedrohung der Ende-zu-Ende-Verschlüsselung wehren. Wir setzen bereits den Perceptual Hash Matching für CSEA-Inhalte in unverschlüsselten Räumen ein und werden weiterhin in diesen Bereich investieren, um schnellere und genauere Abschaltungen zu erreichen und gleichzeitig die Privatsphäre unserer Nutzer zu respektieren.
 
 **Über unsere Arbeit sprechen und unsere Erkenntnisse teilen**  
-Eine berechtigte Kritik an der Stiftung lautet, dass wir nicht viel darüber gesprochen haben, was wir tun, um die Sicherheit der Matrix.org-Nutzer zu gewährleisten, und auch nicht darüber, was wir tun, um sicherzustellen, dass Matrix als Plattform ein sicheres Fundament hat. Das wollen wir im Jahr 2025 ändern, angefangen mit diesem Beitrag.
+Eine berechtigte Kritik an der Stiftung lautet, dass wir nicht viel darüber gesprochen haben, was wir tun, um den Schutz der Matrix.org-Nutzer zu gewährleisten, und auch nicht darüber, was wir tun, um sicherzustellen, dass Matrix als Plattform ein sicheres Fundament hat. Das wollen wir im Jahr 2025 ändern, angefangen mit diesem Artikel.
 
 ## Wie Sie helfen können
 
 ### Als Nutzer von Matrix
 
 Wenn Sie auf Matrix auf schädliche Inhalte stoßen, können Sie diese auf verschiedene Weise melden:
 
-* Wenn Sie ein Matrix.org-Homeserver-Nutzer sind, können [Sie die Inhalte](https://github.com/element-hq/element-x-ios/pull/587) von Ihrem Client aus melden, und sie werden an unser Sicherheitsteam weitergeleitet. Matrix-Clients wie Element iOS ermöglichen es Nutzern, [Räume zu melden](https://github.com/element-hq/element-ios/pull/7805), und wir wissen, dass dies bald auch in Element X, Element Android und Element Web verfügbar sein wird.  
+* Wenn Sie den Matrix.org-Homeserver nutzen, können [Sie die Inhalte](https://github.com/element-hq/element-x-ios/pull/587) von Ihrem Client aus melden, und sie werden an unser Trust & Safety Team weitergeleitet. Matrix-Clients wie Element iOS ermöglichen es Nutzern, [Räume zu melden](https://github.com/element-hq/element-ios/pull/7805), und wir wissen, dass dies bald auch in Element X, Element Android und Element Web verfügbar sein wird.  
```

Comment:
"den" linguistically emphasizes that we mean the single hs that is provided by the Fdn.